app/validators/event.ts

```diff
@@ -34,12 +34,13 @@ export const EditParticipantsValidator = vine.compile(
   vine.object({
     event_code: vine.string().minLength(5).maxLength(5),
     edit_code: vine.string().minLength(5).maxLength(5),
-    remove: vine.boolean(),
-    name: vine.string().maxLength(100),
-    phone_num: vine.string().maxLength(15),
-    gender: vine.enum(['Male', 'Female']),
-    address: vine.string().maxLength(100),
-    can_pickup: vine.boolean(),
-    seats_available: vine.number().withoutDecimals().min(0).max(7),
+    remove: vine.boolean().optional(),
+    name: vine.string().maxLength(100).optional(),
```

Comment:
this isn't ideal because someone could hit the api endpint directly, leaving something like name out, and give null data. fine for the MVP though where we can probably assume there are no bad actors

Reply:
we need to filter the null values at the frontend. it isn't good to ask send the all the values just for updating a single column. also in the frontend these are required. also only developers directly hit the api using postman or something like that. that is why authentication is important